coops-ui/tests/unit/RegisteredOfficeAddress.spec.ts

```diff
@@ -19,7 +19,6 @@ app.setAttribute('data-app', 'true')
 document.body.append(app)
 
 describe('RegisteredOfficeAddress', () => {
-
```

Comment:
Thanks for catching this - I'll check with Serevin about his linter functioning or not.

Reply:
@WalterMoar Yes my linter is functioning, but I have to explicitly call `npm run lint:nofix` before submitting ;)